Frontend/src/pages/ExploreSystems.js

```diff
@@ -486,10 +486,38 @@ function ExploreSystems() {
                                     >
                                         Rational Preperiodic Points
```

Comment:
Could add a comment, listing the sub filters to the rational preperiodic filter.